extension/webpack.config.js

```diff
@@ -3,23 +3,25 @@ const glob = require('glob');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 
-const matchedFiles = glob.sync(`./src/extension/*(*.tsx|*.ts)`, {
+const matchedFiles = glob.sync(`./src/**(extension|components)/*(*.tsx|*.ts)`, {
     nodir: true
 });
 
 const entry = {}
 
 matchedFiles.forEach(file => {
     const TS_FOLDER = path.join(__dirname, 'src/extension');
+    const TSX_FOLDER = path.join(__dirname, 'src/components');
```

Comment:
Is Javascript better for designing popup content UI instead of ts? Was curious why only Popup file is in tsx format. 
Thanks for the explanation in advance :)

Reply:
`ts` and `tsx` are both file extensions for typescript, whereas `js` and `jsx` are their counterparts in javascript.
As for differences between `ts | js` and `tsx | jsx`, the former extensions are simply scripting languages (e.g normal javascript); the latter extensions are combinations of scripting language & HTML. 

In our case, `ts` is regular typescript and `tsx` is typescript with React. :) 
We use React for UI components of Popup so that's the reason for `tsx`

Reply:
@junhur Thanks for the explanation🖤